echopype/convert/set_groups_azfp.py

```diff
@@ -340,6 +340,19 @@ def set_beam(self) -> List[xr.Dataset]:
                         "standard_name": "sound_frequency",
                     },
                 ),
+                # TODO: Add channel dim, to match ek60 and ek80?
+                "beam_type": (
+                    [],
```

Comment:
`beam_type` has a ` channel` dimension in EK60 and EK80. Strictly speaking, it's unnecessary for EK60, right? It's unnecessary for AZFP, for the same reason. I'm hesitant to suggest changes to EK60, in case it has impacts downstream. But then the dimensionality would be inconsistent from the similar case for AZFP, where here I'm adding it as a scalar.

Thoughts.

Reply:
`beam_type` can actually be 1 (split-beam) for EK60, and we do use it for processing in `consolidate.split_beam_angle::get_angle_power_samples`

For AZFP it is unnecessary. Since it has a small footprint (only the `channel` dimension), how about adding that for AZFP and set them all to 0 (single-beam)?

Reply:
I was also  leaning towards adding the `channel` dim in AZFP. Great. I'll change it.

> beam_type can actually be 1 (split-beam) for EK60, and we do use it for processing in consolidate.split_beam_angle::get_angle_power_samples

What I meant in comparing AZFP with EK60 was that in EK60, `beam_type` is the same (1) across channels; or at least that's my understanding!

Reply:
> > beam_type can actually be 1 (split-beam) for EK60, and we do use it for processing in consolidate.split_beam_angle::get_angle_power_samples
> 
> What I meant in comparing AZFP with EK60 was that in EK60, `beam_type` is the same (1) across channels; or at least that's my understanding!

So `beam_type` could change across channel in EK60, depending on the type of transducer installed.

Reply:
Ah, thanks. Good to know.